bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx84.html

```diff
@@ -29,7 +29,7 @@ <h1 class="wnp-main-title">Firefox protects your browsing. <strong>Mozilla VPN</
       <p class="wnp-main-tagline">Mozilla VPN (Virtual Private Network) protects your entire internet connection on your computer, your tablet, and even your phone. For even more security, it comes with a 30-day money back guarantee.</p>
 
       <div class="wnp-main-cta">
-        <a href="https://vpn.mozilla.org/?{{ utm_params }}" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN Now" data-cta-type="button">Get Mozilla VPN Now</a>
+        <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN Now" data-cta-type="button">Get Mozilla VPN Now</a>
```

Comment:
Because the page is now internal, it's okay to drop the `utm_params`, right?

Reply:
Yes we should drop utms now these are same site links 👍

Reply:
> > Quick note – I searched across the code base for all instances of vpn.mozilla.org and found a few. Should this be in a follow-up, low-priority bug? (LMK and I can file)
> 
> I should take care of those here, what did you find?

I found the following that may apply: 
- [ ] [web-eligible-sites.html](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/security/templates/security/bug-bounty/web-eligible-sites.html#L251)
- [ ] [fxa-utm-referral.js](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/media/js/base/fxa-utm-referral.js#L23)
- [ ] [mozilla-fxa-product-button.js](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/media/js/base/mozilla-fxa-product-button.js#L21)

There were a few other links ([1](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L69), [2](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L42), [3](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L42)) but I don't think they apply.